examples/chip-tool/templates/logging/DataModelLogger-src.zapt

```diff
@@ -162,7 +162,7 @@ CHIP_ERROR DataModelLogger::LogEvent(const chip::app::EventHeader & header, chip
 {{/first}}
             case {{asUpperCamelCase parent.name}}::Events::{{asUpperCamelCase name}}::Id:
             {
-                {{zapTypeToDecodableClusterObjectType name ns=parent.name forceNotOptional=true}} value;
+                {{asUpperCamelCase parent.name}}::Events::{{asUpperCamelCase name}}::DecodableType value;
```

Comment:
old generator seems to have been prefixing `chip::app::Clusters::` to this. I would try to keep that the same.

Reply:
I looked at the autogen for commands, and this does the same.

I think we should align with commands instead?
Otherwise I'd have to add in "chip::app::Clusters::" as a prefix which I'd argue isn't necessary (given the compiler is happy).

Can do if you think it's really important - let me know?

Reply:
My reasoning was that adding the prefix makes the delta in DataModelLogger.cpp smaller. Given that we have `using namespace chip::app::Clusters;` at the top of the file it probably does not matter much.